capreolus/extractor/birch_bertpassage.py

```diff
@@ -0,0 +1,71 @@
+import tensorflow as tf
+import numpy as np
+
+from capreolus import get_logger
+from capreolus.utils.exceptions import MissingDocError
+from . import Extractor
+from .bertpassage import BertPassage
+from .common import MultipleTrainingPassagesMixin
+
+logger = get_logger(__name__)
+
+
+@Extractor.register
+class BirchBertPassage(MultipleTrainingPassagesMixin, BertPassage):
```

Comment:
inherite the `create_train_features` and `parse_train_features` from `MultipleTrainingPassagesMixin`, and the other functions from `BertPassage`